src/components/StudentFeedback.vue

```diff
@@ -40,7 +69,12 @@ export default {
     return {
       projects: [],
       module: null,
```

Comment:
Need to remove this. I don't think it is being used anymore.

Reply:
ok cool. maybe we can remove when we're resolving merge conflicts??